draft-regext-rdap-x-media-type.md

```diff
@@ -114,6 +122,15 @@ the resource referenced by the URI is served by a server compliant with this spe
 Otherwise, use of the `application/rdap+json` media type is RECOMMENDED when the URI
 references RDAP resources. 
 
+# RDAP-X Extension
+
+This document defines an RDAP "profile" extension using the identifier "rdapx" (hyphen
+characters are not allowed in RDAP extension identifiers). This RDAP extension defines
+no additional RDAP queries or response structures.
```

Comment:
We could do this later but would need to add this new extension's registration info in the IANA Considerations portion.